core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/UnmappedTerms.java

```diff
@@ -111,10 +97,43 @@ public InternalAggregation doReduce(List<InternalAggregation> aggregations, Redu
 
     @Override
     public XContentBuilder doXContentBody(XContentBuilder builder, Params params) throws IOException {
-        builder.field(InternalTerms.DOC_COUNT_ERROR_UPPER_BOUND_FIELD_NAME, docCountError);
+        builder.field(InternalTerms.DOC_COUNT_ERROR_UPPER_BOUND_FIELD_NAME, 0);
         builder.field(SUM_OF_OTHER_DOC_COUNTS, 0);
         builder.startArray(CommonFields.BUCKETS).endArray();
         return builder;
     }
 
+    @Override
+    protected void setDocCountError(long docCountError) {
+    }
+
+    @Override
+    protected int getShardSize() {
+        return 0;
+    }
+
+    @Override
+    public long getDocCountError() {
+        return 0;
+    }
+
+    @Override
+    public long getSumOfOtherDocCounts() {
+        return 0;
+    }
+
+    @Override
+    protected List<Bucket> getBucketsInternal() {
+        return emptyList();
+    }
+
+    @Override
+    public Bucket getBucketByKey(String term) {
+        return null;
```

Comment:
I'm not sure. If this makes its way back to the user then null seems like the most right thing because that is what mapped terms do when the bucket isn't found. And in this case it is like no buckets are found.